eng/pipelines/coreclr/templates/helix-queues-setup.yml

```diff
@@ -89,17 +89,20 @@ jobs:
     - ${{ if eq(parameters.platform, 'Linux_x64') }}:
       - ${{ if and(eq(variables['System.TeamProject'], 'public'), in(parameters.jobParameters.helixQueueGroup, 'pr', 'ci', 'libraries')) }}:
         - Ubuntu.1804.Amd64.Open
+        - (Centos.Stream9.Amd64.Open)Ubuntu.1804.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:centos-stream9-20220107135047-4cd394c
```

Comment:
This would also duplicate testing on every PR. We usually do horizontal deployment into the platform matrix in our outer loop testing (that being said, I feel like there's nothing running tests on those other queues right now...)